etc/kayobe/kolla.yml

```diff
@@ -147,6 +147,12 @@ kolla_sources:
     type: git
     location: https://github.com/stackhpc/networking-generic-switch.git
     reference: stackhpc/{{ openstack_release }}
+  neutron-server-plugin-networking-mlnx:
+    type: git
+    location: https://opendev.org/x/networking-mlnx
+    # FIXME(wszumski): Last stable release is stable/yoga
+    #reference: stable/{{ openstack_release }}
+    reference: master
```

Comment:
networking-mlnx master has neutron master in requirements.
Consider (untested) https://github.com/stackhpc/networking-mlnx/compare/stackhpc/2023.1-WIP

Reply:
> networking-mlnx master has neutron master in requirements. Consider (untested) https://github.com/stackhpc/networking-mlnx/compare/stackhpc/2023.1-WIP

Good call, I think this is wise since we don't know if they will merge something to master that will break us in the future too.